examples/knative/powershell/kn-ps-twillio-sms/handler.ps1

```diff
@@ -0,0 +1,85 @@
+Function Process-Init {
+   [CmdletBinding()]
+   param()
+   Write-Host "$(Get-Date) - Processing Init`n"
+
+   Write-Host "$(Get-Date) - Init Processing Completed`n"
+}
+
+Function Process-Shutdown {
+   [CmdletBinding()]
+   param()
+   Write-Host "$(Get-Date) - Processing Shutdown`n"
+
+   Write-Host "$(Get-Date) - Shutdown Processing Completed`n"
+}
+
+Function Process-Handler {
+   [CmdletBinding()]
+   param(
+      [Parameter(Position=0,Mandatory=$true)][CloudNative.CloudEvents.CloudEvent]$CloudEvent
+   )
+
+   # Decode CloudEvent
+   try {
+      $cloudEventData = $cloudEvent | Read-CloudEventJsonData -Depth 10
+   } catch {
+      throw "`nPayload must be JSON encoded"
+   }
+
+   try {
+      $jsonSecrets = ${env:SMS_SECRET} | ConvertFrom-Json
+   } catch {
+      throw "`nK8s secrets `$env:SMS_SECRET does not look to be defined"
+   }
+
+   # Check TaskEvent to make sure its VM Create Snapshot Event
+   # See https://williamlam.com/2019/02/creating-vcenter-alarms-based-on-task-events-such-as-folder-creation.html for more details on finding descriptionId from TaskEvent
+   if($cloudEventData.Info.DescriptionId -eq "VirtualMachine.createSnapshot") {
```

Comment:
Perhaps an else or default clause would be neat to log that it skipped this TaskEvent.